x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/locked_card.tsx

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
@parkiino you are likely going to get an eslint error on this license once you merge upstream due the license updates done last night for v2. You can fix it locally by deleting it and then having eslint put it back (or run eslint on your source)